### PR TITLE
Swmaas 730

### PR DESCRIPTION
**Pull request by dnguyen-phunware**
@avakkachen Please review.

### CODE REVIEW

File: Samples/kotlin/src/main/java/com/phunware/kotlin/sample/VoicePromptActivity.kt [hunk 1]
```diff
@@ -205,7 +202,26 @@ class VoicePromptActivity : AppCompatActivity(), OnPhunwareMapReadyCallback,
...
+            val turnManeuver  = pair.turnManeuver
+            var turnable : Boolean  = false
+            if(turnManeuver != null)
+            {
+                turnable = turnManeuver.isTurnManeuver
+            }
+
+            if(turnManeuver != null && position < navigator.maneuvers.size - 1 && turnable)
+            {
+                text += " then "
```
Comment (avakkachen) [inline Samples/kotlin/src/main/java/com/phunware/kotlin/sample/VoicePromptActivity.kt]: Let's move these to strings.xml

File: Samples/kotlin/src/main/java/com/phunware/kotlin/sample/VoicePromptActivity.kt [hunk 2]
```diff
@@ -205,7 +202,26 @@ class VoicePromptActivity : AppCompatActivity(), OnPhunwareMapReadyCallback,
...
+                text += displayHelper.stringForDirection(this, turnManeuver)
+            }
+            else
+            {
+                text += " to arrive at the destination."
```
Comment (avakkachen) [inline Samples/kotlin/src/main/java/com/phunware/kotlin/sample/VoicePromptActivity.kt]: Let's also  make the code formatting java/kotlin specific (braces)

**Comment by avakkachen** [comment]
We might want to merge individual PRs to sample_code_updates till everything is QAed. This one has the target as master.

**Comment by dnguyen-phunware** [comment]
Already merged in #65

**Pull request closed without merging**